golang/vaas/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/GDATASoftwareAG/vaas/golang/vaas
 
-go 1.21
```

Comment:
Please let that stay on 1.21 as this was requested by the outlook team. the problem is, that in the workflow the go-setup does not use the latest version of 1.21 I will have a look at this.